react/src/utils/helpers.ts

```diff
@@ -1,25 +1,50 @@
 export const getISODateTimeFromDateAndTimeString = (date: Date, timeString: string) => {
-  const [hours, minutes] = timeString.split(':').map(Number);
+  const [hours, minutes] = timeString.split(":").map(Number);
   const dateTime = new Date(date);
   dateTime.setHours(hours);
   dateTime.setMinutes(minutes);
   return dateTime;
-}
+};
 
 export const getISODateTimeFromDateAndTime = (date: Date, time: Date) => {
   const dateTime = new Date(date);
   dateTime.setHours(time.getHours());
   dateTime.setMinutes(time.getMinutes());
   return dateTime;
-}
+};
 
 export const weekDayNumberToWeekDayName = (weekDayNumber: number) => {
-  const weekDayNames = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const weekDayNames = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
   return weekDayNames[weekDayNumber];
-}
+};
 
 export const getDateNormalizedDateTime = (dateTime: Date) => {
   const newDate = new Date(dateTime);
-  newDate.setHours(0,0,0,0);
+  newDate.setHours(0, 0, 0, 0);
   return newDate;
-}
+};
+
+export const generateDropappUrl = (
+  oldLink: String,
+  baseId: String | undefined,
+  qrCode: String | undefined,
+  labelIdentifier: String | undefined,
+) => {
+  const newLink = "https://" + oldLink + "?camp=" + baseId + "&preference=classic";
```

Comment:
`oldLink` already contains the protocol prefix, so we don't have to specify it another time. See [this comment](https://github.com/boxwise/boxtribute/pull/466#discussion_r1027113171). Also it then works when running locally (for `REACT_APP_OLD_APP_BASE_URL` I have set `http://localhost:8100`; if https:// is added the link becomes mixed up)

Reply:
I removed the protocol.